docker/.env

```diff
@@ -1,11 +1,2 @@
-HOST_PUDL_IN=./pudl_in
-HOST_PUDL_OUT=./pudl_out
-CONTAINER_HOME=/home/mambauser
-PUDL_INPUT=/home/mambauser/pudl_work/input
-PUDL_OUTPUT=/home/mambauser/pudl_work/output
-DAGSTER_HOME=/home/mambauser/pudl_work/dagster_home
-CONDA_PREFIX=/home/mambauser/env
-PUDL_SETTINGS_YML=/home/mambauser/src/pudl/package_data/settings/etl_full.yml
+PUDL_SETTINGS_YML=/home/mambauser/pudl/src/pudl/package_data/settings/etl_full.yml
 LOGFILE=/home/mambauser/pudl_work/output/pudl-etl.log
-CONDA_RUN="micromamba run --prefix /home/mambauser/env --attach ''"
-GCS_CACHE=gs://zenodo-cache.catalyst.coop
```

Comment:
I removed all the variables which were also being defined inside the Dockerfile to avoid ambiguity. This only left 2 lines. Do we think this file should still exist?  When would we expect the Dockerfile to be falling back on these values? Why shouldn't we be defining them either inside the container, or in the GitHub Actions workflow file?

Reply:
I see no reason for this file to exist anymore, I think it was probably being used in some docker compose situation but we certainly don't seem to use that. We should define the log file in the Dockerfile and the settings location in GHA since it's much more likely we want to change the settings file than the logfile location.

Reply:
I think the `.env` file was only useful when we were debugging this image locally using `docker-compose`. We should probably add the last two variables to the dockerfile.

Reply:
Okay, I'll add them as "set defaults but only if they aren't already set by the script that called the container" variables inside the container.